spec/lib/headlines/security_headers/content_security_policy_spec.rb

```diff
@@ -10,317 +10,42 @@ module SecurityHeaders
       end
 
       describe "#score" do
+        let(:body) { open_fixture("csp/empty_page.html").read }
+
         subject(:score) { described_class.new(headers, body, url).score }
 
-        describe "without header and csp in meta" do
+        context "without header and csp in meta" do
           let(:body) { "" }
           let(:value) { nil }
 
           it { is_expected.to eq(-15) }
         end
 
-        describe "with only header" do
-          let(:body) { open_fixture("csp/empty_page.html").read }
-
-          context "with blank value" do
-            let(:value) { "" }
-
-            it { is_expected.to eq(-15) }
-          end
-
-          context "without valid directives" do
-            let(:value) { "invalid-directive 'self';" }
-
-            it { is_expected.to eq(-15) }
-          end
-
-          context "with at least one valid directive" do
-            let(:value) { "default-src 'self'; invalid-directive 'self';" }
-
-            it { is_expected.to eq(2) }
```

Comment:
We are divide this one large file to smaller files. All of this tests are in different files. But none has been removed.